### ISSUE TITLE
Invalid User-Agent header value

### ISSUE THREAD

**Issue by korlowski** [go-resty/resty#175]
The value of the User-Agent header `go-resty v1.1 - https://github.com/go-resty/resty` does not match the specification (see [RFC 7231](https://tools.ietf.org/html/rfc7231#section-5.5.3)). A valid header should look something like `go-resty/1.1 (https://github.com/go-resty/resty)`

The impact of this is that our akka-http application logs warning logs when resty is calling it:
```
Illegal header: Illegal 'user-agent' header: Invalid input ':', expected OWS, 'EOI', tchar, product-or-comment, comment or ws (line 1, column 27): go-resty v1.1-edge - https://github.com/go-resty/resty
```

**Comment by jeevatkm** [comment]
@korlowski I have updated the UA value format on branch `user-agent-value-format`. Could you try that branch and let me know. Then I will merge it to master and will be released in `v1.9.0`.

**Comment by jeevatkm** [comment]
@korlowski Thanks for the confirmation.
